client/src/pages/guide/english/java/strings/index.md

```diff
@@ -141,3 +141,45 @@ System.out.println(a.compareTo(b));       // 0
 System.out.prinltn(a.compareTo(c));       // 1 since (a>b)
 System.out.println(c.compareTo(a));       // -1 since (c<a)
 ```
+
+#### Splitting Strings  
+If you want to split a string into multiple parts it can easily be done through ```.split()``` this creates an array of the split up parts of the string.
+
+Example of using a delimiter (",") to split a string
+```java
+String text = "Hello, World";
+String[] textParts = text.split(",");
+System.out.println(textParts[0]);
+System.out.println(textParts[1]);
+```
+The result will be:
+
+```
+Hello
+World
```

Comment:
If you `split()` the text `"Hello, World"` on the `,`, then `textPart[1]` will have the space as well before `World`. 

I'll raise a PR to fix this.